src/include/mpir_request.h

```diff
@@ -265,7 +265,7 @@ static inline void MPII_init_request(void)
  * with each pool that they are going to use, typically a 1-1 vci-pool mapping.
  * NOTE: currently, only per-vci thread granularity utilizes multiple request pool.
  */
-static inline void MPIR_Request_registor_pool_lock(int pool, MPID_Thread_mutex_t * lock)
+static inline void MPIR_Request_register_pool_lock(int pool, MPID_Thread_mutex_t * lock)
```

Comment:
nit: unrelated typo fix

Reply:
It must be commited to the wrong patch. I'll check

Reply:
fixed.